exercises/C-comparison-operators/exercise.js

```diff
@@ -5,16 +5,16 @@
   The variables should have values that match the expected results.
 */
 
-var studentCount = 16;
+var studentCount = 15;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = studentCount > mentorCount; // finish this statement
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = roomMaxCapacity == studentCount + mentorCount
 
 var personA = "Daniel";
 var personB = "Irina";
-var sameName; // finish this statement
+var sameName = personA == personB; // finish this statement
```

Comment:
There are two operators for checking equality in JavaScript - "==" and "==="

"==" is loose equality - here JavaScript will convert types when checking
"===" is strict equality - here JavaScript will not convert the types, so is true only if they are the same type.

For example - see what happens if you try `1 == "1"` versus `1 === "1"`